src/ReGenny.cpp

```diff
@@ -1030,10 +1030,13 @@ void ReGenny::update_address() {
 
     if (!m_parsed_address.name.empty()) {
         auto& modname = m_parsed_address.name;
+        std::transform(modname.begin(), modname.end(), modname.begin(), tolower);
```

Comment:
Since you're now modifying `modname` remove the reference so we don't modify the parsed name.

Reply:
Done